test/cli/test_cli_functions.sh

```diff
@@ -284,6 +284,23 @@ function obc_cycle {
     echo_time "✅  obc cycle is done"
 }
 
+function crd_cycle {  
+    local crd 
+    echo_time "Starting the crd cycle"
+    test_noobaa timeout crd create
+    test_noobaa timeout crd wait
+    crd_array=($(kubectl get crd | awk '{print $1}' | grep -v "NAME"))
+    echo_time "${crd_array[*]}"
+    for crd in ${crd_array[@]}
+    do 
+        test_noobaa timeout crd status
+        kuberun describe crd ${crd}     
+    done
+    kuberun get crd
```

Comment:
This command output is:
```
$ kuberun get crd
NAME                                 CREATED AT
backingstores.noobaa.io              2020-05-13T08:15:27Z
bucketclasses.noobaa.io              2020-05-13T08:15:27Z
noobaas.noobaa.io                    2020-05-13T08:15:27Z
objectbucketclaims.objectbucket.io   2020-05-13T08:15:27Z
objectbuckets.objectbucket.io        2020-05-13T08:15:27Z
11:20:22 ✅  kubectl get crd passed
```
or when there is no crd:
```
$ kuberun get crd
No resources found in default namespace.
11:51:07 ✅  kubectl get crd passed
```

we should verify that what we are expecting is there, and run this after each command (create, delete)